src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's better to set the default font styles in the `html` selector using a single `font` shorthand property instead of separate properties for `font-family`, `font-size`, and `font-weight`. This helps to keep the code clean and concise.